style.css

```diff
@@ -65,4 +65,328 @@
     left: 50%;
     transform: translateX(-50%);
     border-radius: 4px;
+}
+
+.header-buttons {
+    text-transform: uppercase;
+}
+
+.book-table {
+    margin-right: 40px;
+}
+
+.book-table,
+.explore {
+    font-size: 16px;
+    font-weight: 400;
+    border: 2px solid #fd5e32;
+    transition-duration: 1.1s;
+    background: transparent;
+    color: white;
+    height: 60px;
+    width: 200px;
+    margin-top: 30px;
+    border: 1px solid rgba(255, 255, 255, 0.5);
+    box-sizing: border-box;
+    border-radius: 5px;
+}
+
+.book-table:hover,
+.explore:hover {
+    background: #E8C300;
+    border-radius: 5px;
+}
+
+.header-left-text {
+    float: left;
+    font-weight: 700;
+    font-size: 12px;
+    line-height: 12px;
+    margin-left: 80px;
+    color: white;
+}
+
+.left-text {
+    transform: rotate(-90deg);
+    position: relative;
+    top: -150px;
+}
+
+.header-right-icons {
+    float: right;
+    margin-right: 80px;
+    position: relative;
+    top: -270px;
+}
+
+.twitter-icon {
+    margin: 20px 0;
+}
+
+.header-arrow {
+    position: relative;
+    margin-top: 60px;
+    padding-bottom: 180px;
+}
+
+.down-btn {
+    position: absolute;
+    left: 50%;
+    transform: translateX(-50%);
+}
+
+.about-section {
+    display: flex;
+    height: 100vh;
+    width: 100%;
+}
+
+.left-side {
+    margin: 217px 80px;
+    text-align: center;
+}
+
+.title {
+    font-size: 36px;
+    line-height: 48px;
+    text-transform: uppercase;
+}
+
+.left-side__line {
+    margin: 10px 0;
+}
+
+.subtitle {
+    font-size: 18px;
+    line-height: 28px;
+    font-weight: 700;
+    margin-bottom: 15px;
+}
+
+.descriptions {
+    font-size: 14px;
+    line-height: 24px;
+    font-weight: 400;
+}
+
+.right-side {
+    position: relative;
+}
+
+.right-side__main-photo {
+    margin: 100px 50px;
+}
+
+.right-side__square {
+    position: absolute;
+    bottom: 200px;
+    right: 0;
+}
+
+.our-team {
+    height: 100vh;
+    width: 100%;
+    color: white;
+    background: url("./images/team-bg.png");
+    background-size: cover;
+}
+
+.our-team__title {
+    text-align: center;
+    font-weight: 700;
+    font-size: 12px;
+    line-height: 100px;
+    letter-spacing: 6px;
+}
+
+.our-team__left-side {
+    position: relative;
+}
+
+.our-team__left-side__square {
+    position: absolute;
+    top: -90px;
+    left: -90px;
+}
+
+.our-team__right-side {
+    margin: 150px 100px 217px 72px;
+    text-align: center;
+}
+
+.sides {
+    display: flex;
+}
+
+.booking {
+    height: 100vh;
+    width: 100%;
+}
+
+.booking__left-side {
+    margin: 140px 40px 0 70px;
+    text-align: center;
+}
+
+.booking__left-side__line {
+    margin-bottom: 20px;
+}
+
+.form__inputs input {
+    background: #E5E5E5;
+    border-radius: 5px;
+    height: 60px;
+    width: 276.8px;
+    border: none;
+    margin-right: 10px;
+    margin-bottom: 10px;
+    padding-left: 15px;
+}
+
+.btn__book-now {
+    position: absolute;
```

Comment:
remove position absolute, it's unnecessary 
and centre button under inputs, now it is moved to the left